### PR TITLE
[READY] Rhino Balance

### PR DESCRIPTION
**Pull request by UnokiAs**
## About The Pull Request

This pr does the following:

- Decrease Armor/Shield of Rhino
- Increase the number of projectile of Rhino guns to 2
- Add new Rhino ert roles
- Access lock the Rhino suits
- Fix the shield not turning off on it's own when destroyed

I wanted also to put a cooldown on the shield for rhinos when it breaks but I have no idea how to do it.
Right now players can technically reopen the shield the moment  it breaks

Solutions for that are the following :
1.  Remove the shield from rhinos (I can do)
2.  Someone find a way to put a cooldown (I can't lol)

## Why It's Good For The Game

Rhino where really tanky and the damage felt sluggish, Now Rhino are less unkillable by abnos and do more damages to them.
Rhino Ert for admins if they want just Rhinos and no rabbits.
Access lock to prevent LC crew to take control of them.

## Changelog
:cl: Unoki
add: Rhino ert
balance: Increase damage output of rhinos / Decrease armor , Only Rabbits and Rhino Unit can drive it
fix: Shield turns off when destroyed
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/ert.dm [hunk 1]
```diff
@@ -480,3 +480,41 @@
...
+	suit = null
+	suit_store = null
+	glasses = /obj/item/clothing/glasses/hud/health/night
+	back = /obj/item/storage/backpack/ert/engineer
+	belt = /obj/item/storage/belt/utility/full
+	l_pocket = null
+	r_pocket = null
+	backpack_contents = list(/obj/item/storage/box/survival/engineer=1)
+
+/datum/outfit/centcom/ert/security/rabbit/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```
Comment (EgorDinamit) [inline code/modules/clothing/outfits/ert.dm:495]:
```suggestion
/datum/outfit/centcom/ert/commander/rhino/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

File: code/modules/clothing/outfits/ert.dm [hunk 2]
```diff
@@ -480,3 +480,41 @@
...
```
Comment (EgorDinamit) [inline code/modules/clothing/outfits/ert.dm:514]:
```suggestion
/datum/outfit/centcom/ert/security/rhino/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

**Comment by UnokiAs** [comment]
Also does team size count the team leader in it, or I should put it to 1 ?

**Comment by EgorDinamit** [comment]
> Also does team size count the team leader in it, or I should put it to 1 ?

Team leader has its own variable in ERT datum.

**Comment by UnokiAs** [comment]
> Team leader has its own variable in ERT datum.

Alright I will set it to 1 to avoid having 3 people when only 2 mechs are here

**Comment by UnokiAs** [comment]
Well nevermind going to leave it in admin can change the number anyways

**Comment by NiftyRyanyan** [comment]
Ok but when will rhinos get rocket launchers for extra friendly fire.

**Pull request merged**